ts/builder.ts

```diff
@@ -574,13 +573,6 @@ export class Builder {
       return this.endVector();
     }
   
-    /**
-     * A helper function to avoid generated code depending on this file directly.
-     */
-    createLong(low: number, high: number): Long {
```

Comment:
I am wondering about people calling this function in user code that will now just get "unknown function" or whatever, that should somehow discover that they should replace this call with a bigint. We could of course leave a version of `createLong` in the code that just produces a bigint, but that would likely leave unnecessary crap code in user code, so maybe it is better to just break them hard? Or is it possible to have use of this function generate a "deprecated: use a bigint instead" warning?

Reply:
If the ambition is to provide a smooth transition we would need to deprecate but still support the old API. That is however a rather large burden and ambition in my opinion and I think we should rather break hard and document it as such (i.e need a semver major verison bump).

Reply:
No, don't want to support the old API anymore, don't worry. I was more wondering if we can guide users better when breakage happens. Like, if we wanted, this function can keep existing and just return bigint, though that would be crufty without a deprecaction warning.

Is this universally supported? https://jsdoc.app/tags-deprecated.html

Reply:
I think the only way worth spending time on is semver and clear release notes. jsdoc is not universally supported and will likely be stripped from different builds etc.

Reply:
Ok, let's then leave it as-is.